dockerfiles/verify-sentry/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM sentry:9.1-onbuild
```

Comment:
we don't need to use the `on-build` variant if we're actually not making any use of it...

Also, I'd probably stay on 9.0 instead of upgrading to 9.1 to make the change smaller.

Reply:
Switched to 9.0.